terraform/modules/jumpbox/main.tf

```diff
@@ -0,0 +1,38 @@
+module "ami" {
+  source = "../common/ami"
+  source_region  = "${var.aws_region}"
+  ubuntu_release = "trusty-14.04-amd64-server"
+  project        = "${var.project}"
+  stack_name     = "${var.stack_name}"
+}
+
+module "ssh_key" {
+  source     = "../common/ssh_key"
+  project    = "${var.project}"
+  stack_name = "${var.stack_name}"
+}
+
+resource "aws_security_group_rule" "allow_ssh" {
+  count             = "${length(var.security_groups)}"
+  security_group_id = "${element(var.security_groups, count.index)}"
+
+  type        = "ingress"
+  from_port   = 0
+  to_port     = 0
+  protocol    = "tcp"
+  cidr_blocks = ["${var.allowed_cidrs}"]
+}
+
+module "jump" {
+  source                      = "../common/instance"
```

Comment:
i suspect our `terraform fmt` travis check is complaining about the alignment of all the `=` here